### ISSUE TITLE
[Chrome] Dev Tools - Network Tab - Load Time

### ISSUE THREAD

**Issue by MichaelThessel** [philc/vimium#2944]
Hi,

Thanks for providing such a great extension. 

I noticed one issue though. In Chrome in the dev tools, when performing network performance profiling on websites the Load time gets messed up when Vimium is enabled. 

![screenshot from 2018-02-06 12-05-18](https://user-images.githubusercontent.com/2926266/35881461-b10bfe38-0b35-11e8-85fa-3da45f8e24aa.png)

This is a problem because I have to disable vimium every time I want to do performance testing.

```
Google Chrome   64.0.3282.119 (Official Build) (64-bit)                             
Revision    5e944b2d79bce5f96dc35a05a76b03d65a6e9a0a-refs/branch-heads/3282@{#589}
OS  Linux                                                                           
JavaScript  V8 6.4.388.40                                                           
Flash   28.0.0.137 /home/removed/.config/google-chrome/PepperFlash/28.0.0.137/libpepflashplayer.so
User Agent  Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/64.0.3282.119 Safari/537.36
Command Line    /opt/google/chrome/google-chrome --flag-switches-begin --flag-switches-end removed 
Executable Path /opt/google/chrome/google-chrome                                    
Profile Path    /home/removed/.config/google-chrome/Default                         
Variations  bd23585d-3f4a17df                                                       
c134752e-b8b72c88                                                                   
3095aa95-3f4a17df                                                                   
d52c4ff7-ca7d8d80                                                                   
47e5d3db-3d47f4f4                                                                   
b1edbc38-cf4f6ead                                                                   
34d450b1-ca7d8d80                                                                   
79616653-3f4a17df                                                                   
f9884634-f23d1dea                                                                   
9e201a2b-6e3ce1c                                                                    
57f575bb-3d47f4f4                                                                   
62c0fa60-ca7d8d80                                                                   
4b61504a-d25ea691                                                                   
9773d3bd-f23d1dea                                                                   
8e3b2dc5-93702590                                                                   
165e16d1-3f4a17df                                                                   
9e5c75f1-1039a221                                                                   
f79cb77b-3d47f4f4                                                                   
4ea303a6-ecbb250e                                                                   
d92562a9-ca7d8d80                                                                   
90bcbadc-3f4a17df                                                                   
25fc488a-4d2fac87                                                                   
58a025e3-c2b41702                                                                   
1bced4a3-90fa85cd                                                                   
b2f0086-93053e47                                                                    
ef25c1eb-3f4a17df                                                                   
2d871858-ca7d8d80                                                                   
494d8760-6843eff2                                                                   
3ac60855-486e2a9c                                                                   
f296190c-861c267                                                                    
4442aae2-a5822863                                                                   
ed1d377-e1cc0f14                                                                    
75f0f0a0-a5822863                                                                   
e2b18481-e1cc0f14                                                                   
e7e71889-4ad60575                                                                   
34baa302-d25ea691                                                                   
41aa6aaa-ca7d8d80                                                                   
94e68624-803f8fc4                                                                   
f141d4bc-28ad44a                                                                    
da4aaa01-4d2fac87 
```

```
Vimium 1.62.4
```

**Comment by gdh1995** [comment]
Repeated on my Vimium 1.62.10, Chrome 64 x64 and Windows 10.

**Comment by luke-denton-aligent** [comment]
I just want to saw me too. Vimium both clears the console of any messages, and resets network timings

**Comment by smblott-github** [comment]
I'm hoping this will be fixed in v1.63, which is on the Chrome Store now.

**Comment by MichaelThessel** [comment]
This has been fixed with v1.63.
